packages/ibm-products/src/components/CreateInfluencer/CreateInfluencer.js

```diff
@@ -43,8 +43,16 @@ export let CreateInfluencer = ({ className, currentStep, stepData }) => {
       getNumberOfDynamicStepsBeforeCurrentStep(stepData, 'shouldIncludeStep') ||
       0;
 
+    const conditionallyRenderInfluencerTitle = () => {
+      if (title) {
+        return <h3 className={cx(`${blockClass}__title`)}>{title}</h3>;
+      }
+      return null;
+    };
+
     return (
       <div className={`${blockClass}__left-nav`}>
+        {conditionallyRenderInfluencerTitle()}
```

Comment:
i don't think a function needs to be declared for this. 
```
{title && <h3 className={`${blockClass}__title`}>{title}</h3>
```
should be sufficient.